src/args.js

```diff
@@ -115,7 +115,9 @@ export const resolveType = (type) => {
         const dictionaryType = { key: resolveType(key), value: resolveType(value) };
         return t.Dictionary(dictionaryType);
       }
-
+      case isPath(type): {
+        return t.Path;
+      }
```

Comment:
```suggestion

      case isPath(type): {
        return t.Path;
      }
      
```

Reply:
Just a bit more space around the case :)

Reply:
sure .

Reply:
> sure .

Oh, come on :)
Not trying to be picky 😛 
It's just one of the things I would personally tinker with 😅